src/components/CardView/CardView.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+
```

Comment:
This newline doesn't make sense if you have  2 imports. With 10 it would make some sense.